client/src/templates/Challenges/utils/frame.ts

```diff
@@ -67,6 +67,10 @@ const createHeader = (id = mainPreviewId) => `
       while(element && element.nodeName !== 'A') {
         element = element.parentElement;
       }
+      if(element && element.nodeName === 'A'){
+        e.preventDefault();
+        window.parent.window.alert('Normally this link would bring you to another website! It works!' + ' This is a link to: ' + '(' + element.href + ')');
```

Comment:
Are we wanting to translate this?

Reply:
Sounds good.  The build error message could use translating, too.